addon/utils/first-load-offline-objects.js

```diff
@@ -0,0 +1,58 @@
+/**
+  @module ember-flexberry-data
+*/
+
+import Ember from 'ember';
+
+/**
+  Load and save offline objects provided by GetJsonForOffline odata function.
+  Tables with objects to load must be empty.
+
+  @function firstLoadOfflineObjects
+  @param {Object} dexieDB Dexie DB (example: dexieService.dexie('DB name', store)).
+  @param {String} odataPath Odata Url (example: 'http://localhost:35904/odata').
+  @param {String} idPath Path to string id in JSON ('guid' by default).
+  @return {Promise}
+*/
+export function firstLoadOfflineObjects(dexieDB, odataPath, idPath = 'guid') {
+  return new Ember.RSVP.Promise(function(resolve, reject) {
+    Ember.$.ajax({
+      method: 'GET',
+      url: `${odataPath}/GetJsonForOffline`
```

Comment:
It seems to be better if we'll move OData function's name to another param of `firstLoadOfflineObjects` function.